--- a/include/target.mk
+++ b/include/target.mk
@@ -13,11 +13,11 @@ __target_inc=1
 DEVICE_TYPE?=router
 
 # Default packages - the really basic set
-DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch libustream-openssl logd urandom-seed urngd
+DEFAULT_PACKAGES:=base-files libc libgcc busybox dropbear mtd uci opkg netifd fstools uclient-fetch libustream-wolfssl logd urandom-seed urngd procd
 # For nas targets
 DEFAULT_PACKAGES.nas:=block-mount fdisk lsblk mdadm
 # For router targets
-DEFAULT_PACKAGES.router:=dnsmasq-full iptables ip6tables ppp ppp-mod-pppoe firewall odhcpd-ipv6only odhcp6c luci luci-lib-ipkg
+DEFAULT_PACKAGES.router:=dnsmasq-full ppp ppp-mod-pppoe firewall odhcpd-ipv6only odhcp6c luci-ssl luci-lib-ipkg
 DEFAULT_PACKAGES.bootloader:=
 
 ifneq ($(DUMP),)